bin/deploy-production-ci.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+# Log into cloud.gov
+cf api api.fr.cloud.gov
+cf login -u $CF_PRODUCTION_SPACE_DEPLOYER_USERNAME -p $CF_PRODUCTION_SPACE_DEPLOYER_PASSWORD -o gsa-opp-analytics -s analytics-dev
```

Comment:
you probably don't want `analytics-dev` as the space

Reply:
agreed. but to address later, as its a larger infrastructure change

https://github.com/18F/analytics-reporter/issues/675